tests/test_utils.py

```diff
@@ -146,3 +151,46 @@ def test_keep_drop_columns(d):
         assert np.all(selected_data[c] == d[c])
     for c in columns[:1]:
         assert c not in selected_data.dtype.names
+
+
+class TestMultiRun(unittest.TestCase):
+    """Test behavior of multi-runs for various different settings."""
+
+    def setUp(self):
+        """Setup context and make some subruns."""
+        self.tempdir = tempfile.mkdtemp()
+        self.context = strax.Context(storage=[strax.DataDirectory(self.tempdir,
+                                                                  provide_run_metadata=True,
+                                                                  readonly=False,
+                                                                  deep_scan=True)],
+                                     register=[Records, Peaks],
+                                     config={'bonus_area': 42, 'use_per_run_defaults': False},
+                                     )
+        self.run_ids = [str(r) for r in range(5)]
```

Comment:
Have you run with more runs? Might be worth to do just  to be sure, especially if the test is fast

Reply:
Is done now in the memory test.